chromeplugin/request.js

```diff
@@ -31,22 +30,15 @@ function detectRedirect(details) {
                + "|(read.amazon.com)"
                + "|(login.amazon.com)"
                + "|(payments.amazon.com)"
-               + "|(amazon.com/clouddrive)";
+               + "|(amazon.com/clouddrive)"
+               + "|(http://)"; //all Amazon pages now redirect to HTTPS, also fixes conflict with HTTPS Everywhere extension

```

Comment:
Should this start with `^` maybe? I don't know if Amazon has that but I'm thinking that there could be cases where another URL is passed as a query parameter or something like that.

Maybe not a big deal.